improver/calibration/reliability_calibration.py

```diff
@@ -34,6 +34,7 @@
 
 import iris
 import numpy as np
+import operator
```

Comment:
Codacy found an issue: [standard import "import operator" should be placed before "import iris"](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6088429)